### PR TITLE
Webdriver manager for opera driver

### PR DESCRIPTION
**Pull request by antoinebou13**
The opera driver is based on the chromedriver, but the driver need opera to be install on the machine to be able to run

[stackoverflow issue](https://stackoverflow.com/questions/52647831/cant-lunch-opera-browser-with-a-selenium-test-with-java-maven-4-0)
[latest operadriver](https://github.com/operasoftware/operachromiumdriver/releases/tag/v.2.45)

When you download the driver and extract the zip it's a directory with sha512 sum with the driver so i had to change the binary file

I tested with the code in the readme

```
from selenium import webdriver
from webdriver_manager.opera import OperaDriverManager

options = webdriver.ChromeOptions()
options.binary_location = "C:\\Users\\USERNAME\\FOLDERLOCATION\\Opera\\VERSION\\opera.exe" # getting error without this line
browser = webdriver.Opera(OperaDriverManager().install() ,options=options)
```

### CODE REVIEW

File: tests/test_opera_manager.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        except:
+            pass
+
+
+def test_operachromium_manager_with_correct_version():
+    driver_path = OperaDriverManager("v.2.45").install()
+    assert os.path.exists(driver_path)
+
+
+def test_operachromium_manager_with_selenium():
```
Comment (SergeyPirogov) [inline tests/test_opera_manager.py]: dummy test?

Comment (antoinebou13) [reply tests/test_opera_manager.py]: I copied the firefox test and test_operachromium_manager_with_selenium only work with a opera binary
I guessed that you didn't have opera install on Travis-CI
I had 6 test pass on my machine

Comment (SergeyPirogov) [reply tests/test_opera_manager.py]: Yes! So we have to add opera installation in travis

Comment (antoinebou13) [reply tests/test_opera_manager.py]: The browser itself not the operachromiumdriver
Maybe you can do it with https://saucelabs.com/
https://docs.travis-ci.com/user/gui-and-headless-browsers/
i don't know much about CI and pytest

**Pull request closed without merging**